ginivision/src/main/java/net/gini/android/vision/camera/CameraFragmentImpl.java

```diff
@@ -390,10 +428,53 @@ private void hideCameraTriggerButtonAnimated() {
         mButtonCameraTrigger.setEnabled(false);
     }
 
-    private void showNoPermissionView() {
-        hideCameraPreviewAnimated();
+    @Override
+    public void showInterface() {
+        if (isNoPermissionViewVisible()) {
+            return;
+        }
+        showInterfaceAnimated();
+    }
+
+    private void showInterfaceAnimated() {
+        showCameraTriggerButtonAnimated();
+        showDocumentCornerGuidesAnimated();
+        if (mImportDocumentButtonEnabled) {
+            showImportDocumentButtonAnimated();
+        }
+    }
+
+    private void showImportDocumentButtonAnimated() {
+        LOG.debug("Showing document import button");
```

Comment:
Don't know if this is really necessary to log into the debug output.

Reply:
I logged for the other hideable interface elements, too, yeah it's not important, but does it hurt? If yes, than I should remove it from the other methods, too. What do you think?

Reply:
I'll go with verbose 🙂